utils/network.go

```diff
@@ -148,6 +148,11 @@ func DoNetworking(args *skel.CmdArgs, conf NetConf, result *current.Result, logg
 			return fmt.Errorf("failed to move veth to host netns: %v", err)
 		}
 
+		err = configureContainerSysctls(hasIPv4, hasIPv6)
```

Comment:
I think this should go above the block above that moves the host end of the veth.  I think we want to make sure it's configured correctly before exposing it.